boards/arm/nrf9160_pca10090/doc/nrf9160_pca10090.rst

```diff
@@ -0,0 +1,206 @@
+is 32 MHz.
+
+Supported Features
+==================
+
+The nrf9160_pca10090 board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| ADC       | on-chip    | adc                  |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| SPU       | on-chip    | system protection    |
++-----------+------------+----------------------+
+| UARTE     | on-chip    | serial               |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+See `nRF91 DK website`_ and `Nordic Semiconductor Documentation library`_
+for a complete list of nRF9160 Development Kit board hardware features.
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED1 (green) = P0.2
+* LED2 (green) = P0.3
+* LED3 (green) = P0.4
+* LED4 (green) = P0.5
+
+Push buttons and Switches
+-------------------------
+
+* BUTTON1 = P0.6
+* BUTTON2 = P0.7
+* SWITCH1 = P0.8
+* SWITCH2 = P0.9
+* BOOT = SW5 = boot/reset
+
+Security components
+===================
+
+- Implementation Defined Attribution Unit (`IDAU`_).  The IDAU is implemented
+  with the System Protection Unit and is used to define secure and non-secure
+  memory maps.  By default, all of the memory space  (Flash, SRAM, and
+  peripheral address space) is defined to be secure accessible only.
+- Secure boot.
+
+
+Programming and Debugging
+*************************
+
+nrf9160_pca10090 supports the Armv8m Security Extension, and by default boots
+in the Secure state.
+
+Building Secure/Non-Secure Zephyr applications
+==============================================
+
+The process requires the following steps:
+
+1. Build the Secure Zephyr application using ``-DBOARD=nrf9160_pca10090`` and
```

Comment:
We probably need a bit more doc about TZ and how you can also combine Zephyr with TF-M. This should probably be elsewhere, somewhere in `doc/`. No need for this to be done here, it can be in a follow-up PR.

Reply:
I agree on this. Perhaps we can do this together with @agross-linaro